static/src/javascripts/projects/common/utils/picturefill.js

```diff
@@ -405,7 +405,7 @@ define([
                 candidates,
                 options = opt || {};
 
-            elements = options.elements || pf.getAllElements;
+            elements = options.elements || pf.getAllElements();
```

Comment:
i'd double check this - we use our own massively edited version (incorporating fandom etc). makes it faster and removes stuff we don't check, but makes it hard to pick up upstream changes.

looks like [our version](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/utils/picturefill.js#L408) doesn't use that?